Packaging.props

```diff
@@ -26,7 +26,7 @@
 
     <!-- We're currently not building a "live" baseline, instead we're using .NETCore 1.0 RTM stable versions as the baseline -->
     <SkipBaseLineCheck>true</SkipBaseLineCheck>
-    <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">2.0.0</PlatformPackageVersion>
+    <MicrosoftNETCorePlatformsPackageVersion Condition="'$(MicrosoftNETCorePlatformsPackageVersion)' == ''">2.0.0</MicrosoftNETCorePlatformsPackageVersion>
```

Comment:
Be careful with this rename.  BuildTools takes a dependency on this property being named `PlatformPackageVersion`.

https://github.com/dotnet/buildtools/blob/0d5177dfbcef15a7d20aaf759e84c421df7216fd/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets#L86

Reply:
Good catch, hadn't considered that.